tests/s3_data_plane_tests.c

```diff
@@ -8114,3 +8114,52 @@ static int s_test_s3_default_get_without_content_length(struct aws_allocator *al
 
     return AWS_OP_SUCCESS;
 }
+
+struct aws_string *test_helper(struct aws_allocator *allocator) {
+    struct aws_string *host_name =
+        aws_s3_tester_build_endpoint_string(allocator, &g_test_public_bucket_name, &g_test_s3_region);
+    return host_name;
+}
+
+AWS_TEST_CASE(test_s3_default_get, aws_test_s3_default_get)
+static int aws_test_s3_default_get(struct aws_allocator *allocator, void *ctx) {
+    (void)ctx;
+
+    struct aws_s3_tester tester;
+    ASSERT_SUCCESS(aws_s3_tester_init(allocator, &tester));
```

Comment:
Variable `hsot_name` has a typo and should be `host_name` for consistency and readability.